models/filter/paymail_filter.go

```diff
@@ -0,0 +1,34 @@
+package filter
+
+// PaymailFilter is a struct for handling request parameters for paymail_addresses search requests
+type PaymailFilter struct {
+	ModelFilter `json:",inline"`
+
+	ID                 *string `json:"id,omitempty"`
+	Alias              *string `json:"alias,omitempty"`
+	Domain             *string `json:"domain,omitempty"`
+	PublicName         *string `json:"publicName,omitempty"`
+	Avatar             *string `json:"avatar,omitempty"`
```

Comment:
### This is rather global comment
Maybe we should put a comment/description and examples on all fields to show them on swagger documentation.

Reply:
ok 👍

Reply:
I've added examples, but when I started adding descriptions it turned out that the comments don't exaplain more than a field name itself so... maybe we should add a comment to only a few fields 🤔

Reply:
ok, let's leave empty descriptions when this is obvious.